templates/huawei-Doradov3/doradoV3Nas.cfg

```diff
@@ -0,0 +1,3211 @@
+PATROL_CONFIG
+"/SENTRY/X/doradoV3Nas/actionList" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/databaseType" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/disabled" = { REPLACE = "0" },
+"/SENTRY/X/doradoV3Nas/embeddedFileList" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/label" = { REPLACE = "Dorado v3 NAS" },
+"/SENTRY/X/doradoV3Nas/macroList" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/messageList" = { REPLACE = "" },
+"/SENTRY/X/doradoV3Nas/parameterList" = { REPLACE = "CollectionErrorCount;1" },
+"/SENTRY/X/doradoV3Nas/productVersion" = { REPLACE = "1000" },
+"/SENTRY/X/doradoV3Nas/runAlertForEachCollectionError" = { REPLACE = "1" },
+"/SENTRY/X/doradoV3Nas/shareCookies" = { REPLACE = "1" },
+"/SENTRY/X/doradoV3Nas/systemType" = { REPLACE = "other" },
```

Comment:
Please fill in the description of the template, with the content of the README.md, thank you!